packages/@react-stately/tooltip/src/useTooltipTriggerState.ts

```diff
@@ -10,23 +10,113 @@
  * governing permissions and limitations under the License.
  */
 
-import {TooltipManager} from './TooltipManager';
-import {useState} from 'react';
+import {TooltipTriggerProps} from '@react-types/tooltip';
+import {useEffect} from 'react';
+import {useId} from '@react-aria/utils';
+import {useOverlayTriggerState} from '@react-stately/overlays';
+
+export const TOOLTIP_DELAY = 1500; // this seems to be a 1.5 second delay, check with design
+export const TOOLTIP_COOLDOWN = 2000;
```

Comment:
I think this should be much lower, maybe like half a second.

Reply:
I'll do it for now, but lets add it to the agenda for tuesday